src/Components/Column/Column.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/react-in-jsx-scope */
```

Comment:
You need to remove this rule from all the files.

Reply:
removed